sdk-dotnet/LittleHorse.Sdk/LHConfig.cs

```diff
@@ -28,6 +28,14 @@ public LHConfig(ILoggerFactory? loggerFactory = null)
             _options = LHOptionsBinder.GetOptionsFromEnvironmentVariables();
             _createdChannels = new Dictionary<string, LittleHorseClient>();
         }
+        
+        public LHConfig(string configOptionsFilePath, ILoggerFactory? loggerFactory = null)
+        {
+            LHLoggerFactoryProvider.Initialize(loggerFactory);
+            _logger = LHLoggerFactoryProvider.GetLogger<LHConfig>();
+            _options = LHOptionsBinder.GetOptionsFromFile(configOptionsFilePath);
```

Comment:
maybe LHConfig is the same that LHOptions

Reply:
I would say to have a private method inside the LHConfig

Reply:
I have 12 configuration options for the client, and they are being used only in the LHConfig file, I can move and remove some other LHConfig properties to make it simpler